doc/source/index.md

```diff
@@ -2,11 +2,19 @@
    :hidden:
 
    Home Page <self>
-   API reference <generated/sleepecg>
+   API reference <api>
    Datasets <datasets>
+   Heartbeat Detection <heartbeat_detection>
    Feature Extraction <feature_extraction>
    Configuration <configuration>
 ```
 
-```{include} ../../README.md
-```
+# SleepECG
+Welcome! This is the documentation for version **{{ version }}** of SleepECG, a toolbox for sleep stage classification using ECG data.
+
```

Comment:
It's probably worth inserting the installation instructions here. Then I would also link to the contributing guide and changelog right here, and only after that I'd list the other items of the website (API, Datasets, ...).